src/web/jcc/client/components/__tests__/Accordion.test.tsx

```diff
@@ -0,0 +1,58 @@
+import { render, screen, waitFor } from "@testing-library/react";
+import { userEvent } from "@testing-library/user-event";
+import { Accordion, AccordionSummary, AccordionDetails, Typography } from "@mui/material";
+
+// Mock the onChange function, so we can track calls to it
+const mockOnChange = jest.fn();
+
+interface Props {
+    onChange: () => void;
+}
+
+const mockProps: Props = { onChange: mockOnChange };
+
+function AccordionTestComponent(props: Props) {
+    return (
+        <Accordion expanded={false} onChange={props.onChange}>
+            <AccordionSummary>
+                <Typography>Accordion Title Here</Typography>
+            </AccordionSummary>
+            <AccordionDetails>Accordion Details Here</AccordionDetails>
+        </Accordion>
+    );
+}
+
+describe("Accordion Tests", () => {
```

Comment:
Nitpicking: What about this for a test name: Accordion expands and collapses on summary click